pyglotaran_extras/types.py

```diff
@@ -10,6 +10,18 @@
 import xarray as xr
 from glotaran.project.result import Result
 
+
+class UnsetType:
+    """Type for the ``Unset`` singleton."""
+
+    def __repr__(self) -> str:  # noqa: DOC
+        """Representation of instances in editors."""
+        return "Unset"
+
+
+Unset = UnsetType()
+"""Singleton to use when arguments are not provided where None is a valid value."""
```

Comment:
The wording is a bit confusing. 

Singleton to use when arguments are not provided, but `None` is also a valid value.

Reply:
reword ok